Package.swift

```diff
@@ -1,10 +1,12 @@
-// swift-tools-version:5.7
+// swift-tools-version:5.10
```

Comment:
If we're bumping to 5.10 we should probably switch on strict concurrency checking

Reply:
Thanks, I'm definitely interested in enabling better checks. How does one turn on the strict concurrency checking?

Reply:
In your target definition in Package.swift add
```swift
swiftSettings: [.enableExperimentalFeature("StrictConcurrency=complete")]
```

Reply:
Thanks, I tried enabling this and there were a non-trivial number of warnings (errors in Swift 6). I've raised an issue #124  to address the concurrency problems and enable the check.